SwiftMessageBar/Message.swift

```diff
@@ -60,6 +62,13 @@ final class Message: UIView {
     let titleLabel = initTitle()
     let messageLabel = initMessage()
     
+    if languageDirection == .rightToLeft {
+      titleLabel.textAlignment = .right
```

Comment:
Well that's a very particular use case, in that case `.natural` won't work of course. Ok, let's review this then :)